From 64c8ef7e0f4304cf7793c8d899701f64bbb9e5f7 Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <cgharzuzi@habana.ai>
Date: Sun, 12 Sep 2021 10:35:18 +0300
Subject: [PATCH] [SW-55618] package: sensors: conf: add label to power

Add label to the power sensors added to hwmon list

Change-Id: I101b317087e046426aca954402933bc11cbbde0a
Signed-off-by: Christine Gharzuzi <cgharzuzi@habana.ai>
---

diff --git a/package/sensors/hl_sensors.conf b/package/sensors/hl_sensors.conf
index c12167b..b66feaf 100644
--- a/package/sensors/hl_sensors.conf
+++ b/package/sensors/hl_sensors.conf
@@ -32,6 +32,8 @@
 label curr2 "iCore"
 label curr3 "ADC 12V input"
 
+label power1 "Power Draw"
+
 ignore in1
 
 chip "HL200-pci-*" "HL202-pci-*" "HL205-pci-*"
@@ -75,6 +77,8 @@
 label curr5 "CPLD 2.5V output"
 label curr6 "Infineon HBM output"
 
+label power1 "Power Draw"
+
 #TODO: to be removed once issue in sensors is fixed.
 ignore curr4
 ignore curr5
